### PR TITLE
fix: recommended products slider

### PR DESCRIPTION
**Pull request by aoltean-plenty**

### CODE REVIEW

File: apps/web/composables/useHomepage/useHomepage.ts [hunk 1]
```diff
@@ -38,6 +42,14 @@ export const useHomepage: UseHomepageDataReturn = () => {
     state.value.loading = false;
   };
 
+  watch(
+    () => currentLocale.value,
+    async (differentLang) => {
+      currentLocale.value = differentLang;
```
Comment (ksted) [inline apps/web/composables/useHomepage/useHomepage.ts]:
```suggestion
    async (newLocale) => {
      currentLocale.value = newLocale;
```
We should use consistent wording, so locale over language here.

I also prefer "new" here to indicate that it's the locale the user switched to.

File: apps/web/composables/useHomepage/useHomepage.ts [hunk 2]
```diff
@@ -23,12 +23,16 @@ export const useHomepage: UseHomepageDataReturn = () => {
...
+      try {
+        await fetchCategoryTemplate(runtimeConfig.public.homepageCategoryId);
+        state.value.data = fetchHomepageTemplate();
+        if (
+          (!state.value.data.blocks || state.value.data.blocks.length === 0) &&
+          state.value.data.meta?.isDefault === null
+        ) {
+          state.value.data = useLocaleSpecificHomepageTemplate(currentLocale.value);
+        }
+      } catch {
```
Comment (ksted) [inline apps/web/composables/useHomepage/useHomepage.ts]: **question**: In which scenario does the try-catch block take over?

**Review by ksted — CHANGES_REQUESTED** [review]

File: apps/web/composables/useHomepage/useHomepage.ts (hunk 1 above)
Comment (aoltean-plenty) [reply apps/web/composables/useHomepage/useHomepage.ts]: I also wanted to go with new but I get a Lint error when using new

Comment (ksted) [reply apps/web/composables/useHomepage/useHomepage.ts]: Looking at the [rule description](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-keyword-prefix.md), I don't think that rationale applies in this case. I think we should `ignore` the rule and go with `newLocale`.

**Review by ksted — APPROVED** [review]

**Pull request merged**